zproject/default_settings.py

```diff
@@ -119,6 +119,7 @@
 SSO_APPEND_DOMAIN: str | None = None
 CUSTOM_HOME_NOT_LOGGED_IN: str | None = None
 
+VIDEO_ZOOM_ACCOUNT_ID = get_secret("video_zoom_account_id", development_only=True)
 VIDEO_ZOOM_CLIENT_ID = get_secret("video_zoom_client_id", development_only=True)
 VIDEO_ZOOM_CLIENT_SECRET = get_secret("video_zoom_client_secret")
```

Comment:
We could instead have different settings for this integration, e.g., `VIDEO_ZOOM_SERVER_TO_SERVER_CLIENT_ID`.

Reply:
I think using the same settings for most of it is probably reasonable. But maybe `VIDEO_ZOOM_SERVER_TO_SERVER_ACCOUNT_ID` would be a good name for that detail, to hint that it's only for that specific variant. Well, it's quite verbose, but I'm not sure we have something shorter than "server_to_server" to clearly identify the new integration.

Reply:
Updated for the more verbose setting name above.